core/src/main/scala/cromwell/core/WorkflowOptions.scala

```diff
@@ -56,7 +56,6 @@ object WorkflowOptions {
   // Misc.
   case object DefaultRuntimeOptions extends WorkflowOption("default_runtime_attributes")
   case object WorkflowFailureMode extends WorkflowOption("workflow_failure_mode")
-  case object Backend extends WorkflowOption("backend")
```

Comment:
Just a heads up, this will break a Centaur test. It will probably have to be re-enabled in the future when we can assign a backend through runtime attributes instead of workflow options.